### PR TITLE
Add version: Microsoft.PowerShell version 7.2.13

### PR DESCRIPTION
**Pull request by anamnavi**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/112150&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/m/Microsoft/PowerShell/7.2.13/Microsoft.PowerShell.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS1.CRLF.7-3-5.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.2.13
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.2.13/Microsoft.PowerShell.installer.yaml:5]:
```suggestion
PackageVersion: 7.2.13.0
```

File: manifests/m/Microsoft/PowerShell/7.2.13/Microsoft.PowerShell.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS1.CRLF.7-3-5.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.2.13
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.2.13/Microsoft.PowerShell.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.2.13.0
```

File: manifests/m/Microsoft/PowerShell/7.2.13/Microsoft.PowerShell.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS1.CRLF.7-3-5.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.2.13
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.2.13/Microsoft.PowerShell.yaml:5]:
```suggestion
PackageVersion: 7.2.13.0
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
PackageVersion should match the DisplayVersion written to Registry that's visible in `appwiz.cpl` / Apps&Features

You would also have to rename the version folder to match the new PackageVersion. The folder under `manifests/m/Microsoft/PowerShell` should also be renamed from `7.2.13` -> `7.2.13.0`

(One quick way to do this is by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online for you to make changes)

File: manifests/m/Microsoft/PowerShell/7.2.13.0/Microsoft.PowerShell.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS1.CRLF.7-3-5.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.2.13
```
Comment (anamnavi) [inline manifests/m/Microsoft/PowerShell/7.2.13.0/Microsoft.PowerShell.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.2.13.0
```

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**